sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -233,12 +228,40 @@ protected ServiceBusClientBuilder getBuilder(boolean useCredentials) {
             .transportType(AmqpTransportType.AMQP)
             .scheduler(scheduler);
 
+        logger.info("Getting Builder using credentials : [{}] ", useCredentials);
         if (useCredentials) {
             final String fullyQualifiedDomainName = getFullyQualifiedDomainName();
 
             assumeTrue(fullyQualifiedDomainName != null && !fullyQualifiedDomainName.isEmpty(),
                 "AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME variable needs to be set when using credentials.");
+            String clientId = System.getenv("AZURE_CLIENT_ID");
+            String domainName = System.getenv("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
```

Comment:
This is already fetched if you look at line 233.

Reply:
This was for debugging in pipeline because It was working on my local but not in pipeline. I will remove this.